packages/editor-ui/src/stores/cloudPlan.store.ts

```diff
@@ -163,6 +168,17 @@ export const useCloudPlanStore = defineStore(STORES.CLOUD_PLAN, () => {
 		window.location.href = `https://${adminPanelHost}/login?code=${code}`;
 	};
 
+	const fetchLeadEnrichmentTemplates = async () => {
+		try {
+			const leadEnrichmentTemplates = await getLeadEnrichmentTemplates(rootStore.getRestApiContext);
+			if (leadEnrichmentTemplates.sections && leadEnrichmentTemplates.sections.length > 0) {
+				useUIStore().setLeadEnrichmentTemplates(leadEnrichmentTemplates);
+			}
+		} catch (error) {
+			console.warn('Error checking for lead enrichment templates:', error);
```

Comment:
please test what happens if network api request fails.. that it falls back gracefully..

Reply:
There is a test for that. If the API call throws, there will be a warning in the console but rest of the app should go on as usual